deluge/core/torrent.py

```diff
@@ -838,9 +841,14 @@ def get_peers(self):
                 client = 'unknown'
 
             try:
-                country = component.get('Core').geoip_instance.country_code_by_addr(
-                    peer.ip[0]
-                )
+                #if it's an I2P peer use I2P logo as flag
```

Comment:
Can simplify this comment to just say 

```suggestion
                # Use I2P logo as country flag
```